File: UMA.App.IdentityManager/Identity/Queries/CreateIdentityQuery.cs
using AutoMapper;
using MediatR;
using UMA.App.Common.AutoMapper;
using UMA.App.IdentityManager.Identity.Models;
using UMA.Domain.Identity;

namespace UMA.App.IdentityManager.Identity.Queries
{
    public class CreateIdentityQuery : IRequest<CreateIdentityResult>, IHaveCustomMapping
    {
        public string Login { get; set; }
        public string Password { get; set; }

        public void CreateMappings(AutoMapper.Profile configuration)
        {
            configuration.CreateMap<CreateIdentityQuery, Domain.Identity.Identity>();
        }
    }
}

File: UMA.Web.Angular/UMA-Web-Angular/src/app/_entities/user/query/createIdentity.query.ts
import { IRequest } from 'src/app/_models/interfaces/IRequest';
import { CreateIdentityViewModel } from '../response/createidentity.viewmodel';

export class CreateIdentityQuery implements IRequest<CreateIdentityViewModel> {
  _response?: CreateIdentityViewModel;
  login: string;
  password: string;
}
